azure-pipelines/build.yml

```diff
@@ -9,7 +9,7 @@ variables:
   value: true
 - name: SignType
   ${{ if ne( variables['Build.Reason'], 'PullRequest' ) }}:
-    value: real
+    value: test
```

Comment:
test in the if and else?

Reply:
Just saw the commit comment about that being temporary

Reply:
Yes, this is to temporarily disable real-signing until everything else works